server/lib/scene/scene.dailyUpdate.js

```diff
@@ -0,0 +1,41 @@
+const schedule = require('node-schedule');
+const { EVENTS } = require('../../utils/constants');
+
+/**
+ * @description Add Schedule job for sunset and sunrise for each house
+ * @returns {Promise} Resolve when success.
+ * @example
+ * scene.dailyUpdate();
+ */
+async function dailyUpdate() {
+  const houses = await this.house.get();
+
+  const jobs = [];
+
+  houses.forEach((house) => {
+    const times = this.sunCalc.getTimes(new Date(), house.latitude, house.longitude);
+
+    jobs.push(
+      schedule.scheduleJob(times.sunrise, () =>
```

Comment:
If scheduleJob is called with a past time, what happens? 

Because Gladys instance may be started at any point during the day, and sunrise could be already in the past.

Reply:
schedule.scheduleJob cannot schedule a job with a past time. this function return 'null'